### PR TITLE
Deduplicate semgrep-langs submodule

### PR DESCRIPTION
**Pull request by mjambon**
PR checklist:
- [x] Documentation is up-to-date
- [x] Changelog is up-to-date
- [x] Change has no security implications (otherwise, ping security team)

### CODE REVIEW

**Comment by IagoAbal** [comment]
I guess this should fix this issue: https://r2c-community.slack.com/archives/CK86BJ5DW/p1642073583056500 (Semgrep not accepting `solidity` as a valid language, whereas Semgrep-core does).

File: .gitmodules [hunk 1]
```diff
@@ -70,9 +70,6 @@
 [submodule "semgrep/lang"]
 	path = semgrep/semgrep/lang
 	url = https://github.com/returntocorp/semgrep-langs
-[submodule "semgrep-core/src/core/ast/lang"]
```
Comment (aryx) [inline .gitmodules:73]: you deleted to create a symlink towards ../../interface/lang which does not exist. It's not gonna work.

**Review by aryx — CHANGES_REQUESTED** [review]

File: .gitmodules (hunk 1 above)
Comment (mjambon) [reply .gitmodules:73]: oops

File: semgrep/tests/e2e/snapshots/test_rule_parser/test_rule_parser__failure__error_messages/badlanguage/error-in-color.txt [hunk 2]
```diff
@@ -5,5 +5,5 @@
 [94m  | [0m               [31m^^^^^^^^^^[0m
 [94m8 | [0m    severity: WARNING
 
-[31munsupported language: intercal. supported languages are: bash, c, c#, c++, cpp, csharp, generic, go, golang, hack, hcl, html, java, javascript, js, json, kotlin, kt, lua, none, ocaml, php, py, python, python2, python3, r, regex, ruby, rust, scala, sh, terraform, tf, ts, typescript, vue, yaml[0m
+[31munsupported language: intercal. supported languages are: bash, c, c#, c++, cpp, csharp, docker, dockerfile, generic, go, golang, hack, hcl, html, java, javascript, js, json, kotlin, kt, lua, none, ocaml, php, py, python, python2, python3, r, regex, ruby, rust, scala, sh, sol, solidity, terraform, tf, ts, typescript, vue, yaml[0m
```
Comment (aryx) [inline semgrep/tests/e2e/snapshots/test_rule_parser/test_rule_parser__failure__error_messages/badlanguage/error-in-color.txt:8]: yep, this fix the solidity issue :)

**Pull request merged**